docs/install.rst

```diff
@@ -6,8 +6,8 @@ Installing pg_auto_failover
 We provide native system packages for pg_auto_failover on most popular Linux
 distributions.
 
-Use the steps below to install pg_auto_failover on PostgreSQL 16. At the
-current time pg_auto_failover is compatible with PostgreSQL 13 to 16.
+Use the steps below to install pg_auto_failover on PostgreSQL 17. At the
+current time pg_auto_failover is compatible with PostgreSQL 13 and newer.
```

Comment:
I think I prefer the spelling “PostgreSQL 13 to 17”, because soon enough PostgreSQL 18 will be out.

Reply:
Changed in 50051c6